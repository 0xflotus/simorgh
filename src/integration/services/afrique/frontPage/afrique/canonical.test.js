/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service afrique
 * @pathname /afrique
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical afrique frontPage', runCanonicalTests);
