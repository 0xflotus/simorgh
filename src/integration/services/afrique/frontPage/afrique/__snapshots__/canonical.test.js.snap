// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical afrique frontPage A11y I can see the skip to content link 1`] = `"Aller au contenu"`;

exports[`Canonical afrique frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=3&amp;p=afrique.page&amp;x1=[urn:bbc:cps:36f554aa-fa63-58e7-e050-380a4e80102b]&amp;x2=[responsive]&amp;x3=[news-afrique]&amp;x4=[fr]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[Accueil+-+BBC+News+Afrique]&amp;x11=[1970-01-17T23:43:18.842Z]&amp;x12=[1970-01-19T02:08:51.005Z]\\"/>"`;

exports[`Canonical afrique frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/afrique"`;

exports[`Canonical afrique frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. La BBC n'est pas responsable du contenu des sites externes. Découvrez notre approche en matière de liens externes."`;

exports[`Canonical afrique frontPage Header I can see the branding 1`] = `"BBC News, Afrique"`;

exports[`Canonical afrique frontPage I can see Index Alsos 1`] = `"Lire plus"`;

exports[`Canonical afrique frontPage SEO Canonical link 1`] = `"http://localhost:7080/afrique"`;

exports[`Canonical afrique frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical afrique frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical afrique frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical afrique frontPage SEO Heading level 1 1`] = `"BBC News, Afrique - Accueil"`;

exports[`Canonical afrique frontPage SEO Lang attribute 1`] = `"fr"`;

exports[`Canonical afrique frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/afrique",
    "@type": "WebPage",
    "name": "Accueil",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
      "width": 1024,
    },
    "name": "BBC News Afrique",
    "publishingPrinciples": "https://www.bbc.com/afrique/institutionelles-49283281",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
  "url": "http://localhost:7080/afrique",
}
`;

exports[`Canonical afrique frontPage SEO OG description 1`] = `"BBC Afrique, le service en langue française de la BBC, vous offre toute l'actualité en Afrique et dans le monde. Politique, sport, culture, articles, photos, vidéo et radio. Une offre multimédia complète: une fenêtre sur l'Afrique et sur le monde."`;

exports[`Canonical afrique frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png"`;

exports[`Canonical afrique frontPage SEO OG image alt 1`] = `"BBC News Afrique"`;

exports[`Canonical afrique frontPage SEO OG locale 1`] = `"fr"`;

exports[`Canonical afrique frontPage SEO OG site name 1`] = `"BBC News Afrique"`;

exports[`Canonical afrique frontPage SEO OG title 1`] = `"Accueil - BBC News Afrique"`;

exports[`Canonical afrique frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical afrique frontPage SEO OG url 1`] = `"http://localhost:7080/afrique"`;

exports[`Canonical afrique frontPage SEO Page title 1`] = `"Accueil - BBC News Afrique"`;

exports[`Canonical afrique frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical afrique frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical afrique frontPage SEO Twitter creator 1`] = `"@bbcafrique"`;

exports[`Canonical afrique frontPage SEO Twitter description 1`] = `"BBC Afrique, le service en langue française de la BBC, vous offre toute l'actualité en Afrique et dans le monde. Politique, sport, culture, articles, photos, vidéo et radio. Une offre multimédia complète: une fenêtre sur l'Afrique et sur le monde."`;

exports[`Canonical afrique frontPage SEO Twitter image alt 1`] = `"BBC News Afrique"`;

exports[`Canonical afrique frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png"`;

exports[`Canonical afrique frontPage SEO Twitter site 1`] = `"@bbcafrique"`;

exports[`Canonical afrique frontPage SEO Twitter title 1`] = `"Accueil - BBC News Afrique"`;
